src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -71,9 +71,9 @@ class ErrorHandler
     private $loggers = array(
         E_DEPRECATED => array(null, LogLevel::INFO),
         E_USER_DEPRECATED => array(null, LogLevel::INFO),
-        E_NOTICE => array(null, LogLevel::NOTICE),
-        E_USER_NOTICE => array(null, LogLevel::NOTICE),
-        E_STRICT => array(null, LogLevel::NOTICE),
+        E_NOTICE => array(null, LogLevel::WARNING),
+        E_USER_NOTICE => array(null, LogLevel::WARNING),
```

Comment:
@Tobion: with the reasoning at https://github.com/symfony/symfony/pull/14181#issuecomment-89052687, I'd keep the warning log level for user notices. Ok for you?
I'll wait for your opinion before merging.